docs/technical-setup/install-a-git-client.md

```diff
@@ -15,14 +15,58 @@ It is therefore a user-friendly option which does not require you to use the com
 
 GitKraken is _free_ to use with public repositories like `OpenScPCA-analysis`; you do not need to upgrade to their paid plan!
 
-### Install GitKraken
+Be aware that installing GitKraken will install Git, but _only_ for use in GitKraken.
+If you want to also be able to use Git via command line, please refer to the [command line installation instructions](#why-use-the-git-command-line-interface).
 
-1. Install GitKraken [using this link](https://www.gitkraken.com/download). 
+### Install GitKraken on macOS
+
+1. Install GitKraken [using this link](https://www.gitkraken.com/download).
 Note that installing GitKraken will also provide you with Git itself.
 
 1. Set up GitKraken on your machine by [directly signing in with your GitHub account](https://help.gitkraken.com/gitkraken-client/github-gitkraken-client/#sign-in-with-github).
 This will automatically provide you with the credentials you need to interact with GitHub without further setup.
 
+### Install GitKraken on Windows with WSL2
+
+!!! note
+    Before installing GitKraken, make sure your WSL2 installation is up to date by running this command in Windows Powershell (not in Ubuntu).
+    When launching Powershell, make sure you are [running as an administrator](https://learn.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7.4#with-administrative-privileges-run-as-administrator).
+
+    ```sh
+    wsl --update
+    ```
+
+    Still need to install WSL2?
+    [Follow these instructions first](./install-wsl2.md).
+
+You will [install GitKraken as a Linux application](https://help.gitkraken.com/gitkraken-client/how-to-install/#deb) into the WSL2 side of your computer.
+
+All of the following installation steps are commands that you should run in the Ubuntu terminal.
+
+
+1. Download the GitKraken Linux installer:
+
+    ```sh
+    wget https://release.gitkraken.com/linux/gitkraken-amd64.deb
+    ```
+
+2. Install GitKraken:
+
+    ```sh
+    sudo apt install ./gitkraken-amd64.deb
+    ```
```

Comment:
I wonder if we want to also tell people to remove these files after finishing? Or we should perhaps tell people to download to a subfolder to keep the home directory a bit cleaner?